src/Integration/Persistence/FileBindingSerializer.cs

```diff
@@ -29,26 +29,22 @@
 
 namespace SonarLint.VisualStudio.Integration.Persistence
 {
+    /// <summary>
+    /// Writes the binding configuration file to the source controlled file system
+    /// </summary>
+    /// <remarks>
+    /// The file will be enqued but not actually written.
+    /// It is the responsibility of the caller to flush the queue.
+    /// This is to allow multiple other files to be written using the 
+    /// same instance of the SCC wrapper (e.g. ruleset files).
+    /// </remarks>
```

Comment:
An earlier version of the new connected mode needed the ability to commit a single file change directly to disk using the ISourceControlledFileSystem (I think it was when the only file being written was the config file).
That code is no longer used so I've removed it.